### ISSUE TITLE
How to Develop Real-time Apps with MeteorJS

### ISSUE THREAD

**Issue by raghunath-r-a** [LoginRadius/engineering-portal#757]

**Comment by saisumanthkumar** [comment]
@raghunath-r-a I'd like to work on this topic,please assign me

**Comment by raghunath-r-a** [comment]
Hi @saisumanthkumar 

This is now assigned to you. Please proceed with an outline and then drafting the blog post.

If you're stuck anywhere or having doubts, please do let us know.

**Comment by saisumanthkumar** [comment]
can we add the resources ??

**Comment by raghunath-r-a** [comment]
> can we add the resources ??

@saisumanthkumar I can't understand what you're specifically asking. Would you please explain a bit elaborately?

**Comment by saisumanthkumar** [comment]
Inaddition to this ,can I add some tutorial links(Youtube)

**Comment by raghunath-r-a** [comment]
@saisumanthkumar Yes, you can add 1 or 2 at max. But please make sure the youtube link is highly relevant to the blog posts and ensure that you embed the video, not just adding a link.

**Comment by saisumanthkumar** [comment]
ok 👍🏻
